### PR TITLE
Opt out several tag filters

### PR DESCRIPTION
**Pull request by liangxia**
Examples pass [log](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-4.15-upgrade-from-stable-4.13-gcp-ipi-basecap-none-f14/1773141876094799872/artifacts/gcp-ipi-basecap-none-f14/cucushift-upgrade-prepare/build-log.txt),
```
/verification-tests /
+ cucumber --tags '@4.14 and @amd64 and @gcp-ipi and @network-ovnkubernetes and not @fips and not @storage and not @storage and not @console and not @console and not @storage and not @customer
        and not @destructive
        and not @flaky
        and not @inactive
        and not @prod-only
        and not @qeci
        and not @security
        and not @stage-only
 and @upgrade-prepare' -p junit
Using the junit and dir_embedder profiles...
```

/cc @pruan-rht @jianlinliu @jhou1

### CODE REVIEW

**Comment by jianlinliu** [comment]
several 4.15 chain upgrade ci yaml is being bumped to 4.16 in https://github.com/openshift/release/pull/50345, I guess we  can leave the change in 4.15 chain upgrade ci yaml to @JianLi-RH, so that avoid some git conflicts.

**Comment by liangxia** [comment]
> several 4.15 chain upgrade ci yaml is being bumped to 4.16 in #50345, I guess we can leave the change in 4.15 chain upgrade ci yaml to @JianLi-RH, so that avoid some git conflicts.
> 
> Or hold this PR for a while until PR#50345 get merged (I guess it is close to be merged), then rebase, and update all of them together.

Sure. Restored all the changes for chainupgrade.

**Comment by jianlinliu** [comment]
@liangxia https://github.com/openshift/release/pull/50345 get merged, do you mind to rebase, and update other ci jobs together?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml [hunk 1]
```diff
@@ -514,8 +512,8 @@ tests:
     cluster_profile: ibmcloud-qe-2
     env:
       TEST_FILTERS: ~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
-      UPGRADE_CHECK_RUN_TAGS: not @fips and @disconnected
-      UPGRADE_PRE_RUN_TAGS: not @fips and @disconnected
+      UPGRADE_CHECK_RUN_TAGS: '@disconnected'
+      UPGRADE_PRE_RUN_TAGS: '@disconnected'
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml:516]: Shall we also remove the console tag on https://github.com/openshift/release/blob/e8eedfc0df5c362db263ad1ef58ddf65b9fd89fc/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml#L145 ?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.14.yaml [hunk 2]
```diff
@@ -307,8 +307,6 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       BASELINE_CAPABILITY_SET: None
       PERSISTENT_MONITORING: "false"
-      UPGRADE_CHECK_RUN_TAGS: not @console and not @storage
-      UPGRADE_PRE_RUN_TAGS: not @console and not @storage
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.14.yaml:311]: Shall we also remove the console tag on https://github.com/openshift/release/blob/e8eedfc0df5c362db263ad1ef58ddf65b9fd89fc/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.14.yaml#L151 ?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml [hunk 3]
```diff
@@ -267,8 +267,6 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       BASELINE_CAPABILITY_SET: None
-      UPGRADE_CHECK_RUN_TAGS: not @console and not @storage
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml:270]: Shall we also remove the console tag on https://github.com/openshift/release/blob/e8eedfc0df5c362db263ad1ef58ddf65b9fd89fc/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml#L139 ?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml [hunk 4]
```diff
@@ -136,8 +136,8 @@ tests:
       BASELINE_CAPABILITY_SET: None
       PERSISTENT_MONITORING: "false"
       TEST_FILTERS: ~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
-      UPGRADE_CHECK_RUN_TAGS: '@disconnected and not @console and not @storage'
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml:139]: Shall we also remove the console tag on https://github.com/openshift/release/blob/e8eedfc0df5c362db263ad1ef58ddf65b9fd89fc/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml#L151 ?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml (hunk 1 above)
Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml:516]: @jianlinliu We can detect the cluster is fips or not for all clusters to determine whether we should run fips test. And detect the basecap-none clusters to determine whether we should run console/storage tests.
For the one you linked `aws-ipi-disc-priv-sno-f28`, I don't have the background why it excludes console tests. So I just leave it as it is now.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml:516]: Got it.

**Pull request merged**